pkg/server/events.go

```diff
@@ -57,9 +57,13 @@ func newEventMonitor(c *containerstore.Store, s *sandboxstore.Store) *eventMonit
 	}
 }
 
-// subscribe starts subsribe containerd events. We separate subscribe from
+// subscribe starts subscribe containerd events. We separate subscribe from
```

Comment:
My initial comment is incomplete... Please remove the second sentence.

Reply:
done